### ISSUE TITLE
simulateur d’empreinte carbone du ferry

### ISSUE THREAD

**Issue by pibies** [laem/futureco-data#122]
Ce simulateur prends il en compte le fret? Quand je prends le ferry (sans cabine et sans voiture) il y a aussi une dizaine de camions de marchandises qui font le voyage. Ce n'est pas le cas pour un voyage en avion. Est ce que la fonction de transport de marchandises du ferry ne devrait pas entrer en compte et réduire mon empreinte carbone (et aussi celle de celui qui voyage avec son camping car) ?
PS: sure cette page https://hackmd.io/@laem/empreinte-ferry il est bien précisé que le modèle choisi est celui du volume mais je ne trouve pas la réponse à la question du transport de marchandise.
2em commentaire: J'ai un peu de mal avec la simulation qui plus que double mon empreinte carbone sur mon Marseille Ajaccio juste parce que je prends un repas ou une bière dans le bateau. Je pense qu'il faudrait lisser le calcul dans la fonction consommation de services.
NB: le calcul est évidemment très complexe donc merci pour le boulot ! Ca faisait longtemps que je cherchais ce simulateur pour ferry et maintenant il existe!

**Comment by laem** [comment]
> Ce simulateur prends il en compte le fret

Oui. Le volume occupé par le passager est divisé par un volume total qui comprend celui utilisé par le fret. 

>  et réduire mon empreinte carbone

C'est le cas :) 

> Ce n'est pas le cas pour un voyage en avion

Il y a des chances que si :)

> plus que double mon empreinte carbone sur mon Marseille Ajaccio juste parce que je prends un repas ou une bière dans le bateau. Je pense qu'il faudrait lisser le calcul dans la fonction consommation de services.

Oui, l'idée serait de pouvoir dire ce qu'on consomme comme services. Tout le monde n'utilise pas la piscine, la boutique, etc.
